secret_service/secret_service.go

```diff
@@ -67,11 +69,35 @@ func (s *SecretService) OpenSession() (dbus.BusObject, error) {
 	return s.Object(serviceName, sessionPath), nil
 }
 
+// GetCollections returns list of available collections
+func (s *SecretService) CheckCollectionPath(path dbus.ObjectPath) (bool, error) {
```

Comment:
I think this can be simplified to just return an error in case the path is not found.

Also the doc comment is wrong :)

Reply:
-> it has to start with the function name for godoc:

// CheckCollectionPath ...